programStructure/Chessboard.js

```diff
@@ -16,29 +16,25 @@ Passing this string to console.log should show something like this:
 When you have a program that generates this pattern, define a binding size = 8 
 and change the program so that it works for any size, outputting a grid of the given width and height.
 */
-let size = 8;
-for(let i = 0; i < size; i++)
-{
-  let stringForChess = "";
-  for(let k = 0; k < size; k++)
-  {
-    if(i % 2 === 0)
-    {
-      if(k % 2 === 0)
-      {
-      	stringForChess = stringForChess + " ";
+function generateChessboard(size) {
+  for (let i = 0; i < size; i++) {
+    let stringForChess = "";
+    for (let k = 0; k < size; k++) {
+      if (i % 2 === 0) {
+        if (k % 2 === 0) {
+          stringForChess = stringForChess + " ";
+        } else {
+          stringForChess = stringForChess + "#";
+        }
+      } else {
+        if (k % 2 === 0) {
+          stringForChess = stringForChess + "#";
+        } else {
+          stringForChess = stringForChess + " ";
+        }
       }
```

Comment:
You have two almost identical block statement here:
` if (k % 2 === 0) {
    stringForChess = stringForChess + "#";
  } else {
    stringForChess = stringForChess + " ";
  }
`. Is it possible to change your code in a way when one block with small modification replace two blocks? Try to find similarity and differences between them. Then you can merge two blocks in to one